test/test.js

```diff
@@ -219,7 +219,7 @@ describe('combobox-nav', function() {
 
       assert.equal(options[1].getAttribute('aria-selected'), 'true')
       assert.equal(input.getAttribute('aria-activedescendant'), 'hubot')
-      assert.equal(list.scrollTop, 36)
+      assert.equal(list.scrollTop, options[1].offsetTop)
```

Comment:
this test was failing ... it seems that the height of the list items can very by platform. This removes the magic number.